### ISSUE TITLE
Could this repo achieving a good performance on VOC?

### ISSUE THREAD

**Issue by BucolicWind** [eriklindernoren/PyTorch-YOLOv3#559]
I use this repo on VOC,but I cannot get a good performance,and I train my own dataset, the same problem occurred?How could I solve this problem? help thanks

**Comment by cwq159** [comment]
I've tried VOC07+12. The results are as follows:
(VOC)YOLOv3: 42.63%
(VOC)YOLOv3-Tiny: 30.47%
Compared to COCO2014:
(COCO)YOLOv3: 54.93%
(COCO)YOLOv3-Tiny: 30.87%
It seems that the results of VOC are poor than COCO. But it maybe due to the hyperparameters. Hope someone can achieve higher mAP by adjusting the hyperparameters. Welcome to discuss with me.

**Comment by summersdog** [comment]
> I've tried VOC07+12. The results are as follows:
> (VOC)YOLOv3: 42.63%
> (VOC)YOLOv3-Tiny: 30.47%
> Compared to COCO2014:
> (COCO)YOLOv3: 54.93%
> (COCO)YOLOv3-Tiny: 30.87%
> It seems that the results of VOC are poor than COCO. But it maybe due to the hyperparameters. Hope someone can achieve higher mAP by adjusting the hyperparameters. Welcome to discuss with me.

could u please share your hyperparameters?I have trained on voc07 for 100 epochs but got only 0.18 mAP

**Comment by cwq159** [comment]
font{
        line-height: 1.6;
    }
    ul,ol{
        padding-left: 20px;
        list-style-position: inside;
    }


    
    
        Do u adjust your anchor size for VOC? In fact, I haven’t changed the hyperparameters yet.
    
        
            
        
    
    
      
    
        
                    
            
                
                    
                            
                                
                            
                            
                                Weiqian Chen
                            
                            
                                
                            
                    
                        
                            
                                    cwq@pku.edu.cn
                            
                        
                
            
        
        
    
    
        签名由
        网易邮箱大师
        定制
    
 
    

    On 12/5/2020 20:07，byy<notifications@github.com> wrote： 




I've tried VOC07+12. The results are as follows:
(VOC)YOLOv3: 42.63%
(VOC)YOLOv3-Tiny: 30.47%
Compared to COCO2014:
(COCO)YOLOv3: 54.93%
(COCO)YOLOv3-Tiny: 30.87%
It seems that the results of VOC are poor than COCO. But it maybe due to the hyperparameters. Hope someone can achieve higher mAP by adjusting the hyperparameters. Welcome to discuss with me.

could u please share your hyperparameters?I have trained on voc07 for 100 epochs but got only 0.18 mAP

—You are receiving this because you commented.Reply to this email directly, view it on GitHub, or unsubscribe.

**Comment by DeidaraYang** [comment]
Hi, have you achieved better performance right now?

**Comment by dog-qiuqiu** [comment]
me too
